### DISCUSSION TITLE
Mística Bootcamp

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#504] [Proposals]
Sería muy interesante plantear sesiones para aquellos nuevos diseñadores que entren en la compañía o empiecen a trabajar con Mística en donde se les explique cómo usar Mística de la mejor forma posible.

Podrían ser vídeos en Loom

Retos:
- Cómo evitar mantener un video y lo que se cuenta, es decir, que no se quede obsoleto. Quizás hay que plantear el contenido de la manera más agnóstica posible.
- Cómo saber cuando un diseñador/a entra en Telefónica para que puede acceder a este contenido? (Es necesario saber esto?)
- Añadir a success factor el perfil de design system como área de conocimiento?

Otro tipo de materiales también puede ser interesante para este Bootcamp como una especie de Quiz
https://www.figma.com/file/PlHcR3HP6bqCStrqbjGVPp/M%C3%ADstica-Training

**Comment by aweell** [comment]
Me parece una idea fantástica, quizás aquí se podría añadir un módulo con lo básico en el uso de Playroom enlazando con lo que comentas en https://github.com/Telefonica/mistica-design/discussions/512?

**Reply by yceballost** [reply]
Completamente!

**Comment by yceballost** [comment]
Cómo podría vivir esto en Brand Factory?
Me imagino dentro del Get started haya una página de Onboarding y aquí te permita entrar a una especie de "codecademy" de Mística? (Mística Academy) 😁

**Comment by yceballost** [comment]
https://github.com/Telefonica/mistica-design/issues/2118
